src/main/resources/headers/cublas_header.h

```diff
@@ -0,0 +1,16 @@
+#ifndef CUBLAS_HEADER_H
+#define CUBLAS_HEADER_H
+
+#include <cublas_v2.h>
+
+#define CUBLAS_CALL(f)                                           \
+  {                                                              \
+    cublasStatus_t err = (f);                                    \
+    if (err != CUBLAS_STATUS_SUCCESS) {                          \
+      fprintf(stderr, "CUBLAS error occurred\n");                \
+      exit(err);                                                 \
+    }                                                            \
+  }                                                              \
+
+
+#endif
```

Comment:
I know it sounds silly but GitHub doesn't like it if the end of files are not newline. 
When that happens, you see a circle with minus at the bottom of the file.

Reply:
I checked on the server. 

I'll fix the format :)